src/Components/Server/ref/Microsoft.AspNetCore.Components.Server.netcoreapp3.0.cs

```diff
@@ -92,6 +91,13 @@ public partial class RemoteUriHelper : Microsoft.AspNetCore.Components.UriHelper
         public static void NotifyLocationChanged(string uriAbsolute) { }
     }
 }
+namespace Microsoft.AspNetCore.SignalR
+{
+    public partial class ComponentEndpointConventionBuilder : Microsoft.AspNetCore.SignalR.HubEndpointConventionBuilder
```

Comment:
FYI I chose to not seal this because you can't have sealed type as a generic argument - https://stackoverflow.com/questions/1945085/why-cant-we-use-sealed-classes-as-generic-constraints

It makes sense if the type never changes, but not allowing the type as a generic parameter doesn't allow for the type one day changing to unsealed.

Reply:
@rynowak Change sealed builders at https://github.com/aspnet/AspNetCore/pull/9657 to unsealed? An internal constructor achieves the same result.

Reply:
that would be fine.

Reply:
@JamesNK can't have sealed as a generic argument? you surely meant generic constraint? (upper bound constraint)

Reply:
I think you're right @NinoFloris. I don't know why this can't just be sealed.

If someone was planning to add `T : ComponentEndpointConventionBuilder` as a generic constraint and ComponentEndpointConventionBuilder is sealed, you could just use ComponentEndpointConventionBuilder directly instead of introducing T.

Reply:
OK after some discussion: https://github.com/aspnet/AspNetCore/issues/9316

Could you please seal this and the regular HubConventionBuilder and add an inferface?